api/src/context.rs

```diff
@@ -118,13 +113,21 @@ impl Context {
         }
     }
 
-    pub fn get_latest_ledger_info_poem(&self) -> AptosInternalResult<LedgerInfo> {
-        self.get_latest_ledger_info().map_err(|e| {
-            AptosErrorResponse::InternalServerError(Json(
-                AptosError::new(format_err!("Failed to retrieve ledger info: {}", e).to_string())
-                    .error_code(AptosErrorCode::ReadFromStorageError),
+    // TODO: Add error codes to these errors.
+    pub fn get_latest_ledger_info_poem<E: InternalError>(&self) -> Result<LedgerInfo, E> {
+        if let Some(oldest_version) = self.db.get_first_txn_version().map_err(E::internal)? {
```

Comment:
Why did the error codes go away?

Reply:
Oop they don't need to, I can add them back.